### PR TITLE
issue: 755772 configure time check for mlx5_hw features in MLNX_OFED

### PR DESCRIPTION
**Pull request by rosenbaumalex**
Signed-off-by: Alex Rosenbaum <Alexr@mellanox.com>

### CODE REVIEW

**Comment by rosenbaumalex** [comment]
@igor-ivanov please review

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/345/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/363/ for details (Mellanox internal link).

File: src/vma/hw/mlx5/wqe.h [hunk 1]
```diff
@@ -15,21 +15,12 @@
...
-	uint32_t	rsvd0;
-	uint8_t		cs_flags;
-	uint8_t		rsvd1;
-	uint16_t	mss;
-	uint32_t	rsvd2;
-	uint16_t	inline_hdr_sz;
-	uint8_t		inline_hdr_start[2];
-	uint8_t		inline_hdr[16];
-};
+#endif //DEFINED_MLX5_HW_ETH_WQE_HEADER
```
Comment (igor-ivanov) [inline src/vma/hw/mlx5/wqe.h]: What is reason of removing it?
What target ofed version is it for?

Comment (rosenbaumalex) [inline src/vma/hw/mlx5/wqe.h]: `struct mlx5_wqe_eth_seg` is not used by VMA in any way.
Maybe it was there when coding started but not any more.

Comment (igor-ivanov) [inline src/vma/hw/mlx5/wqe.h]: I see it at https://github.com/Mellanox/libvma/blob/experimental_256B/src/vma/hw/mlx5/wqe.h#L39

Comment (rosenbaumalex) [inline src/vma/hw/mlx5/wqe.h]: thanks. I missed that.
I did not check on older MLNX_OFED to verify it works.
I'll push new patch....

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/379/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/361/ for details (Mellanox internal link).

**Comment by rosenbaumalex** [comment]
@igor-ivanov, @serglelv, any additional feedback?

**Comment by igor-ivanov** [comment]
:+1:

**Pull request merged**